src/services/safe-wallet-provider/rpc.ts

```diff
@@ -0,0 +1,35 @@
+import { type Eip1193Provider } from 'ethers'
+import { type SafeWalletProvider } from '.'
+
+export class SafeRpcProvider implements Eip1193Provider {
+  walletProvider: SafeWalletProvider
+
+  constructor(walletProvider: SafeWalletProvider) {
+    this.walletProvider = walletProvider
+  }
+
+  async request(request: { method: string; params?: Array<any> | Record<string, any> }): Promise<any> {
+    if (request.method === 'eth_requestAccounts') {
+      request.method = 'eth_accounts'
+    }
+    const response = await this.walletProvider.request(0, request, {
+      id: -1,
+      url: '',
+      name: 'Built-inswap',
+      description: 'description',
+      iconUrl: 'icon',
+    })
+
+    if ('result' in response) {
+      return response.result
+    }
+  }
+
+  async on() {
+    console.log('swap on =============================')
```

Comment:
Useful logs are OK to keep, but please make them look like the other logs, e.g. the GA ones.

Reply:
This class is not used at all. We forgot to remove it.